src/assisted_test_infra/test_infra/tools/load_system.py

```diff
@@ -0,0 +1,163 @@
+from abc import ABC, abstractmethod
+from string import Template
+
+from service_client import log
+
+
+class Command(ABC):
+    base_command = None
+    redirect = " > /dev/null &"
+
+    def __init__(self, nodes: list):
+        """Initialize node_undo dict, keys are node object
+        :param nodes: node is the command receiver  run_command
+        """
+        self.node_undo_commands = dict()
+        self._nodes = nodes
+
+    @property
+    def receivers(self):
+        # check if nodes exist and active before running command
+        active_nodes = []
+        for node in self._nodes:
+            if self._verify_node_active(node):
+                active_nodes.append(node)
+        return active_nodes
+
+    @staticmethod
+    def _verify_node_active(node):
+        if node and node.is_active:
+            return node
+        log.info(f"Node {node.name} not active anymore")
+        return None
+
+    @abstractmethod
+    def execute(self):
+        pass
+
+    def undo(self):
```

Comment:
just in general, it's better to use two terms that are opposites of each other (to make the API clearer)
something like `start` and `stop` instead of `execute` and `undo`